cratetorrent/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+jobs = 1
```

Comment:
Why is this needed?

Reply:
I didn't mean to include that, I usually build with 1 thread since I'm doing homework or playing games while I program.